File: storage/internal-storage/src/main/java/cz/cuni/mff/socneto/storage/internal/data/model/Job.java
package cz.cuni.mff.socneto.storage.internal.data.model;

import lombok.Data;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import java.util.Date;
import java.util.UUID;

@Data
@Entity
public class Job {
    @Id
    private UUID jobId;
    private String jobName;
    @Column(name = "userId")
    private String username;
    private String topicQuery;
    private String status;
    private String language;
    private Date startedAt;
    private Date finishedAt;
}

File: backend/Domain/Models/Storage/Response/Job.cs
using System;
using Socneto.Domain.Models.JMS.Response;

namespace Socneto.Domain.Models.Storage.Response
{
    public class Job
    {
        public Guid JobId { get; set; }
        public string JobName { get; set; }
        public string Username { get; set; }
        public string TopicQuery { get; set; }
        public JobStatusCode Status { get; set; }
        public string Language { get; set; }
        public DateTime StartedAt { get; set; }
        public DateTime? FinishedAt { get; set; }
    }
}